Paralogues.pm

```diff
@@ -0,0 +1,700 @@
+=head1 LICENSE
+
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2023] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=head1 CONTACT
+
+ Ensembl <http://www.ensembl.org/info/about/contact/index.html>
+
+=cut
+
+=head1 NAME
+
+ Paralogues
+
+=head1 SYNOPSIS
+
+ mv Paralogues.pm ~/.vep/Plugins
+
+ # Download Ensembl paralogue annotation (if not available in current directory)
+ # and fetch variants from Ensembl API (requires database access)
+ ./vep -i variations.vcf --plugin Paralogues
+
+ # Download Ensembl paralogue annotation (if not available in current directory)
+ # and fetch variants from custom VCF file
+ ./vep -i variations.vcf --plugin Paralogues,vcf=/path/to/file.vcf
+
+ # Fetch Ensembl variants in paralogue proteins using only the Ensembl API
+ # (requires database access)
+ ./vep -i variations.vcf --database --plugin Paralogues,mode=remote
+
+=head1 DESCRIPTION
+
+ A VEP plugin that fetches variants overlapping the genomic coordinates of amino
+ acids aligned between paralogue proteins. This is useful to predict
+ pathogenicity of variants in paralogue positions.
+
+ This plugin automatically downloads paralogue annotation from Ensembl databases
+ if not available in the current directory (by default). Use argument `dir` to
+ change the directory of the paralogue annotation:
+   --plugin Paralogues
+   --plugin Paralogues,dir=/path/to/dir
+
+ It is also possible to point to a custom tabix-indexed TSV file by using
+ argument `paralogues`:
+   --plugin Paralogues,paralogues=file.tsv.gz
+   --plugin Paralogues,dir=/path/to/dir,paralogues=paralogues_file.tsv.gz
+   --plugin Paralogues,paralogues=/path/to/dir/paralogues_file.tsv.gz
+
+ The overlapping variants can be fetched from Ensembl API (by default) or a
```

Comment:
As far as I understand to fetch from the Ensembl API we have to run the plugin connecting to the database but if we want to run in offline mode we have to use a VCF file?
Running the command:
`--plugin Paralogues,paralogues=annotation/Paralogues.pm_homo_sapiens_110_GRCh38.tsv.gz`
I get the warning:
`WARNING: Failed to instantiate plugin Paralogues: ERROR: Cannot fetch Ensembl variants in offline mode; please define vcf argument in the Paralogues plugin`
This means I have either to 1) connect to the db or 2) input a vcf file?
In the instructions, it could mention if we need to connect to the db or not.

Reply:
Added paragraph to description to make it clearer. Thanks!